cli/internal/runsummary/execution_summary.go

```diff
@@ -89,6 +89,16 @@ func (ts *TaskExecutionSummary) MarshalJSON() ([]byte, error) {
 	return json.Marshal(&serializable)
 }
 
+// ExitCode access exit code nil means no exit code was received
+func (ts *TaskExecutionSummary) ExitCode() *int {
+	var exitCode int
+	if ts.exitCode == nil {
+		return nil
+	}
+	exitCode = *ts.exitCode
+	return &exitCode
+}
```

Comment:
Is this different from just  `return ts.exitCode?` (in which case do you just want to make the field public?)

Reply:
No, it should be the same. I was just trying to change as few lines as possible with this PR.